game/action.go

```diff
@@ -42,27 +42,27 @@ func (a *Action) ShowTo(p *Player) string {
 	case Play:
 		if a.WithKicker {
 			if a.Target == nil {
-				return fmt.Sprintf("%d: %s with kicker", a.Card.Kicker.CastingCost.Colorless, a.Card)
+				return fmt.Sprintf("%s: %s with kicker", a.Card.Kicker.CastingCost.Colorless, a.Card)
```

Comment:
I think this should be passing just `a.Card.Kicker.CastingCost` rather than with the `.Colorless` on the end? If it is the one with `.Colorless` then the format should be a `%d`